app/models/service/aggregation.rb

```diff
@@ -42,26 +151,36 @@ def aggregate_direct_vm_memory_on_disk
   end
 
   def aggregate_all_vm_cpus
-    all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
+    attributes["aggregate_all_vm_cpus"] || all_vms.inject(0) { |aggregate, vm| aggregate += vm.cpu_total_cores.to_i }
```

Comment:
Gotcha.  If it is making a hash everytime, then yeah, maybe the other route is preferred.  Will investigate.